tests/virtualfolder_helpers.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
```

Comment:
Perhaps, but I just copied this from other file in tests. I checked 5 of them randomly and they all have this line. Doesn't harm anyway.